src/main/java/de/tum/in/www1/artemis/domain/Submission.java

```diff
@@ -76,6 +76,13 @@ public abstract class Submission extends DomainObject implements Comparable<Subm
     @Column(name = "submission_date")
     private ZonedDateTime submissionDate;
 
+    /**
+     * This field is set directly via repository @Query. It read only by the client.
+     */
+    @SuppressWarnings("unused")
+    @Column(name = "plagiarism_suspected")
+    private Boolean plagiarismSuspected;
```

Comment:
This information is needed in the client every time information about submission or participation is displayed to the student. 

Latest submission entry is always already fetched in such situations, so no additional queries and REST calls are necessary to get cpc information.

Storing this flag in `PlagiarismSubmission` entries may be inefficient due to the following:
- `PlagiarismSubmission`s are not fetched by default in most pages, so additional REST call and database queries would be necessary,
- for such call new endpoints should be introduced,
- `PlagiarismSubmission` doesn't have any constraint defined with `Submission` so formally no joins should be made between them.

cc: @krusche

Reply:
Agreed not to proceed with this flag. We will create plagiarism cases with special messages instead